server/src/test/java/org/cloudfoundry/identity/uaa/alias/EntityAliasHandlerValidationTest.java

```diff
@@ -124,16 +124,10 @@ final void shouldReturnFalse_NotBothAliasPropsEmptyInReqBody() {
             // (8) alias ID removed, alias ZID changed
             requestBody = buildEntityWithAliasProps(null, "some-other-zid");
             assertThat(aliasHandler.aliasPropertiesAreValid(requestBody, existingEntity)).isFalse();
-        }
 
-        @Test
-        final void shouldReturnTrue_BothAliasPropsEmptyInReqBody() {
-            final T existingEntity = buildEntityWithAliasProps(
-                    UUID.randomUUID().toString(),
-                    CUSTOM_ZONE_ID
-            );
-            final T requestBody = buildEntityWithAliasProps(null, null);
-            assertThat(aliasHandler.aliasPropertiesAreValid(requestBody, existingEntity)).isTrue();
+            // (9) alias ID removed, alias ZID removed
+            requestBody = buildEntityWithAliasProps(null, null);
+            assertThat(aliasHandler.aliasPropertiesAreValid(requestBody, existingEntity)).isFalse();
```

Comment:
Not that it really matters to me, but just curious. Is there any specific reason that we use `org.assertj.core.api.Assertions` instead of `org.junit.jupiter.api.Assertions` in this class?